lisa/tools/python.py

```diff
@@ -53,7 +55,10 @@ def dependencies(self) -> List[Type[Tool]]:
     def _install(self) -> bool:
         package_name = "python3-pip"
         assert isinstance(self.node.os, Posix)
-        self.node.os.install_packages(package_name)
+        if isinstance(self.node.os, BSD):
+            self.node.os.install_packages("py39-pip")
+        else:
+            self.node.os.install_packages(package_name)
```

Comment:
```
if isinstance(self.node.os, BSD):
    package_name = "py39-pip"
self.node.os.install_packages(package_name)
```

Reply:
updated the if statement to minimize the code size.